.travis.yml

```diff
@@ -1,37 +1,9 @@
-language: php
+version: ~> 1.0
 
-dist: xenial
-
-services:
-  - mysql
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/self-jobs-range.yml
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=2.x-dev
-
-matrix:
-  include:
-    - php: 7.1
-      env: DB=MYSQL
-    - php: 7.2
-      env: DB=MYSQL
-    - php: 7.2
-      env: DB=MYSQL
-    - php: 7.3
-      env: DB=MYSQL
-
-before_script:
-  - phpenv rehash
-  - phpenv config-rm xdebug.ini
-
-  - composer validate
-  - composer require --no-update silverstripe-themes/simple ~3.2.0
-  - if [[ $DB == PGSQL ]]; then composer require silverstripe/postgresql:^2 --no-update; fi
-  - composer install --prefer-source --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
-
-  # Validate cow schema
-  - composer global require silverstripe/cow ^2
-  - ~/.config/composer/vendor/bin/cow schema:validate
```

Comment:
@emteknetnz I think the Cow schema validation needs to be reintroduced? Looks like `COW_TEST=1` is the way to do this based on [shared config](https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/base.yml#L55).

Reply:
COW_TEST is included in the shared jobs matrix the `self-jobs-range` provision uses -  https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/jobs/self-range.yml#L12

Reply:
Ah, missed that sorry.